pkg/k8sutils/templates/pod/spec.go

```diff
@@ -81,6 +69,19 @@ func getProbe(port int32, path string) corev1.ProbeHandler {
 	}
 }
 
+func completeProbe(failureSeconds *int32, defaultFailureThreshold int32, defaultPeriodSeconds int32,
+	probeHandler corev1.ProbeHandler) *corev1.Probe {
+	probe := &corev1.Probe{}
+	if failureSeconds != nil {
+		probe.FailureThreshold = (*failureSeconds) / defaultPeriodSeconds
```

Comment:
what if failureSeconds can't be divided by defaultPeriodSeconds, e.g. provided failureSeconds is 1, and defaultPeriodSeconds is 5, should the `FailureThreshold` evaluated to `0` or `1`?

Reply:
FailureThreshold will use be the default value, which is 3.